os_migrate/playbooks/delete_conversion_hosts.yml

```diff
@@ -15,8 +15,8 @@
   tasks:
     - name: unregister the conversion hosts
       ansible.builtin.include_role:
-        name: os_migrate.os_migrate.conversion_host
-        tasks_from: unregister.yml
+        name: os_migrate.os_migrate.conversion_host_content
```

Comment:
We should have these tasks as part of the same role, I believed we had it that way

Reply:
+1 good catch